arroyo/processing/strategies/decoder/json.py

```diff
@@ -1,26 +1,72 @@
-from typing import Optional
+import logging
+import time
+from typing import Mapping, Optional
 
 import fastjsonschema  # pip install sentry-arroyo[json]
 import rapidjson
 
 from arroyo.processing.strategies.decoder.base import Codec, ValidationError
+from arroyo.utils.metrics import get_metrics
+
+logger = logging.getLogger(__name__)
 
 
 class JsonCodec(Codec[object]):
-    def __init__(self, json_schema: Optional[object] = None) -> None:
+    def __init__(
+        self,
+        json_schema: Optional[object] = None,
+        json_schema_name: Optional[str] = None,
```

Comment:
nit: I'd skip this. We already tag the metrics backend with the storage/consumer group/topic before passing it to Arroyo, so I don't think there's too much ambiguity about which consumer any metrics are coming from.